Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -78,11 +77,6 @@ def shutdown(self):
 
         self.get_session().stop_upnp()
 
-        # Save DHT state
```

Comment:
@whirm the DHT state is part of the libtorrent state since 0.15. I've changed it in commit 267d66a4ada606d6443bf08f13f538d663a9df58.